### DISCUSSION TITLE
Docker from NAS

### DISCUSSION THREAD

**Discussion by Shyriu59** [robske110/IDDataLogger#43] [Q&A]
Hi everyone

1st of all, great job on the data logger , it looks real good .
Quite the beginner so I was wondering if there was an easy way to install indocker on a Synology NAS or on a VM (Buster ) that I already run for jeedom to not multiply systems and maintenance… yeah lazy me…
Thanks for your insights

**Comment by robske110** [comment]
It is certainly possible to install IDDataLogger directly using docker on a Synology NAS, however it is not very beginner-friendly. A good reference would be http://uitechmind.com/2022/01/05/how-do-i-use-docker-compose-with-synology/, but for IDDataLogger quite a few steps do differ.

I think installing in your existing VM via docker is probably the easiest way. I recommend using docker in the VM, because the Jeedom installation will most likely interfere with the default install script in this repo.
